src/styles/styles.js

```diff
@@ -2878,6 +2878,48 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    footerWrapper: {
+        backgroundColor: themeColors.midtone,
+        fontSize: variables.fontSizeNormal,
+        paddingTop: 64,
+        paddingLeft: 17,
+    },
+
+    footerColumnsContainer: {
+        maxWidth: 1100,
+        flex: 1,
+        flexWrap: 'wrap',
+        marginBottom: 40,
+    },
+
+    footerColumn: {
+        padding: 15,
+    },
+
+    footerTitle: {
+        fontSize: variables.fontSizeLarge,
+        color: themeColors.success,
+        marginBottom: 16,
+    },
+
+    footerRow: {
+        paddingVertical: 4,
+        marginBottom: 8,
+        color: themeColors.textLight,
+        fontSize: variables.fontSizeMedium,
```

Comment:
Not sure what this value is, but ideally the size is 15.

Reply:
Here's the font size variables for reference! I'll switch to normal in that case
<img width="351" alt="image" src="https://user-images.githubusercontent.com/38015950/218579481-255c471e-266e-473a-a285-3a8fe083af4a.png">

Reply:
Perfect, thanks! Also, I wonder where fontSizeMedium is even used? I feel like we should get rid of it, but probably in a different PR.